microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -281,18 +281,113 @@ DevTools now tracks all of the objects that were GC'd during the recording.  Use
 
 If you are investigating objects that were only GC'd during specific major or minor GC operations, configure the settings appropriately to track the operation you care about. To learn more about the differences between major and minor GC, see [Trash talk: the Orinoco garbage collector | V8 JavaScript engine developer blog](https://v8.dev/blog/trash-talk).
 
+
 <!-- ====================================================================== -->
-## Spot frequent garbage collections
+## Find DOM tree memory leaks from detached elements
 
-If your page appears to pause frequently, then you may have garbage collection issues.
+A DOM node is only garbage-collected when there are no references to the node from either the DOM tree or JavaScript code running on the page.  A node is said to be "detached" when it is removed from the DOM tree but some JavaScript still references it.  Detached DOM nodes are a common cause of memory leaks.
 
-You can use either the Microsoft Edge Browser Task Manager or Performance memory recordings to spot frequent garbage collection.
 
-*  In the Microsoft Edge Browser Task Manager, frequently rising and falling **Memory** or **JavaScript Memory** values represent frequent garbage collection.
+<!-- ------------------------------ -->
+#### Tools for investigating detached elements
+
+| Tool | Pros | Cons | Docs |
+|---|---|---|---|
+| **Memory** tool > **Detached elements** option button (profiling type) | Shows elements as DOM tree nodes. | Doesn't link to JavaScript source code. | [Debug DOM memory leaks ("Detached elements" profiling type)](./dom-leaks-memory-tool-detached-elements.md) |
+| **Memory** tool > **Heap snapshot** option button (profiling type) > **Detached** | The detached element has a link to its JavaScript source code.<!-- todo: how get to JS? --> | Shows elements as objects in memory. | [Debug DOM memory leaks ("Detached elements" profiling type)](./dom-leaks-memory-tool-detached-elements.md) |
```

Comment:
is it true that the displayed detached element has a link to its JavaScript source code, when using **Memory** tool > **Heap snapshot** option button (profiling type) > type **Detached** in the **Filter by class** text box?

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/?branch=pr-en-us-3282#tools-for-investigating-detached-elements

![image](https://github.com/user-attachments/assets/1278be16-04d6-4677-8a5e-005d2fb55895)